ocaml/typing/printtyp.ml

```diff
@@ -1126,6 +1127,13 @@ let rec tree_of_typexp mode ty =
               when Path.same path Predef.path_option ->
                 tree_of_typexp mode ty
             | _ -> Otyp_stuff "<hidden>"
+          else if is_position l then
+            match get_desc (tpoly_get_mono ty1) with
+            | Tconstr(path, [], _)
+              when Path.same path Predef.path_lexing_position ->
```

Comment:
Is this check necessary? I would think it wouldn't be.

Reply:
Maybe not, but I think it's analogous to the check above for optional arguments?
Either way, I realize I don't need this bit of code at all - if we're generating an unused type, we might as well let that be done by the `else` case below. The casing is only necessary for option because we want to directly print the child node